server/src/main/java/org/activityinfo/ui/client/page/entry/location/LocationSearchPresenter.java

```diff
@@ -94,10 +94,10 @@ private void numberLocations(List<LocationDTO> locations) {
         int number = 0;
         for (LocationDTO location : locations) {
             if (location.hasCoordinates()) {
-                location.setMarker(String.valueOf((char) ('A' + number)));
                 number++;
+                location.setMarker(number.toString());
```

Comment:
I don't think two digits will fix but you can check, or perhaps we can use different shape markers that allow for more space... I'm certainly open to anything that works!